java-checks/src/test/files/filters/LombokFilter.java

```diff
@@ -244,6 +244,31 @@ private UtilityClassWithPublicConstructorCheck() {
   }
 }
 
+class LombokVal {
```

Comment:
would be nice to add test which is using `import lombok.val` and just `val` as a type

Reply:
I truly don't know how the front-end will react on this... knowing that val is supported with java 11. I fear to open the Pandora Box. But I'm going to have a look.